include/configs/sama5d3_xplained.h

```diff
@@ -59,13 +60,14 @@
 
 /* SPL */
 #define CONFIG_SPL_TEXT_BASE		0x300000
-#define CONFIG_SPL_MAX_SIZE		0x18000
+#define CONFIG_SPL_MAX_SIZE			0x18000
```

Comment:
Why is this change here? Anything I am missing? Can you restore this line to the original if nothing changed ?

Reply:
👍